Makefile

```diff
@@ -94,7 +94,11 @@ protoset:
 
 test:
 	@cd ./web; go test
+	@cd ./web/hooks; go test
```

Comment:
I think these can all be replaced with a single line:
```
go test ./...
```
Or
```
go test -v ./...
```

Reply:
Yeah, -v is just too vocal. Changed to `go test ./...`.